src/plugin.ts

```diff
@@ -629,13 +629,23 @@ function updateCallForm<EnvSpecificOptions>(
           ),
       ])
     );
+
     // we just wrapped the target callExpression in the call to
     // setComponentTemplate. Adjust `target` back to point at the
     // precompileTemplate call for the final updateScope below.
     //
     target = target.get('arguments.0') as NodePath<t.CallExpression>;
   }
 
+  // Changing imports/identifiers doesn't automatically update the scope.
+  //
+  // NOTE: https://github.com/babel/babel/issues/7974
+  //
+  // We need to refresh ourselves
+  // This is expensive, but required so that maybePruneImport
+  // has a chance to prune imports when their specifiers are no longer used.
+  state.program.scope.crawl();
```

Comment:
Ah, so the previous release *just* cleaned up all our need for scope.crawl. It appears that this one place was not covered.

The fix for this will go into babel-import-util. Using `util.removeImport` is supposed to correct the scope references.

Reply:
Looking closer I didn't explain correctly in my comment above, but the basic point is still true: we don't need scope.crawl when we're the ones directly responsible for removing the reference. We should be updating the scope at that point so it stays correct.